CK2Plus_expanded/history/provinces/1468 - Fayoum.txt

```diff
@@ -0,0 +1,16 @@
+# 1468 - Fayoum

```

Comment:
Sorry, @daniloy, looking at this again:

This province doesn't seem to match up with vanilla. 1468 is Leh in vanilla. What's going on here?

Reply:
It looks like there are some other issues of this variety. What do we do here?